nanohttp/validation.py

```diff
@@ -95,6 +95,21 @@ def create_exception(self):
 # noinspection PyAbstractClass
 class RequiredValidator(Criterion):
 
+    def __init__(self, expression):
+        if isinstance(expression, str):
+            error = expression
+        else:
+            error = '400 Bad request'
```

Comment:
This is so implicit, Current users may stay not aware of what happened.

Please raise an error when the expression is not either bool and or string.